sestest.ps1

```diff
@@ -2,7 +2,7 @@
 # Convenience Launcher for sestest
 #
 
-$sestest = "$PSScriptRoot\out\sestest\Debug\netcoreapp1.1\sestest.dll"
+$sestest = "$PSScriptRoot\out\sestest\netcoreapp2.0\sestest.dll"
```

Comment:
sestest.dll doesn't target netstandard?

Reply:
xUnit requires the test assembly to target netcoreapp.